src/component/material-ui/nav-wrapper/index.js

```diff
@@ -87,25 +128,25 @@ class NavWrapper extends Component {
 
     const loginSignupJSX = (
       <RaisedButton
-        label="Signup/Login"
+        className="login-logout-button"      
         onClick={(event) => this.setState({
           anchorEl: event.target,
           popoverOpen: true,
         })}
-      />
+      >
```

Comment:
Why did you change this? My understanding was that with Material UI you weren't supposed to put the names inside of the element, you were supposed to use the `label` instead. This should be switched back. (At least that's what Josh said [I think])

Reply:
Well we did it this way in nearly everything else, and didn't use the label or primaryText prop, so i made it consistent this way with what we've done everywhere else.

we can go back through and fix this with another cleanup/branch, i'll add a TODO for that.